### ISSUE TITLE
Numbers of barcode Get wrong

### ISSUE THREAD

**Issue by eddyeHa** [serratus/quaggaJS#219]
![codebar issues](https://user-images.githubusercontent.com/28991447/30033882-b5b0b51a-9163-11e7-8e16-942ec103968f.png)
Hi I have some issues whit the codebar-reader in live stream.

The  live stream reader throw me a wrong number.

Its there a way I can handle the live stream reader.

Sorry me English. Your Library is great!!
![img_20170904_111839](https://user-images.githubusercontent.com/28991447/30033850-7c96f384-9163-11e7-9b69-a74dbb04faa6.jpg)

**Comment by andreabioli** [comment]
I have the same problem: I understand that it depends on my poor webcam, but a false positive is always a big problem... Isn't there anything we can do?

**Comment by eddyeHa** [comment]
Hi, I have a clue of how this working:
first you need to know what type of code are you reading, for example in mi case the type code is an "ean reader"

here I let you al the codes that quagga works:
![imagen](https://user-images.githubusercontent.com/28991447/30767020-e4545962-9fbe-11e7-86c5-494380bc0d03.png)

I recommend you only use one code type.

also you need a perfect Ilumination.

If you had more questions or doubts, ask me... i apologize again for my Englis

**Comment by andreabioli** [comment]
Yes, I tried every combination, but the results are always changing, for the same barcode, at least for me...

**Comment by lakshman2891** [comment]
hello @eddyeHa
i have the same issue, are you able to solve the issue??

**Comment by lakshman2891** [comment]
Hello everyone!
i used the below order-
decoder: {
        readers: [
            'code_128_reader',
            'ean_8_reader',
            'upc_reader',
            'upc_e_reader'
]

This is the order i used and it recognizes all the mentioned code with less false positives. but if i add a new barcode type then its having problem. 
note: try to work with single type of barcode.
